src/settings.h

```diff
@@ -1094,6 +1094,8 @@ namespace Settings
                 { ItemDefinitionIndex::WEAPON_KNIFE, { ItemDefinitionIndex::WEAPON_KNIFE_M9_BAYONET, -1, -1, -1, -1, -1, "" } },
                 { ItemDefinitionIndex::GLOVE_CT_SIDE, { ItemDefinitionIndex::GLOVE_SPECIALIST, 10006, 0.0005f, -1, -1, -1, "" } },
                 { ItemDefinitionIndex::GLOVE_T_SIDE, { ItemDefinitionIndex::GLOVE_STUDDED_BLOODHOUND, 10006, 0.0005f, -1, -1, -1, "" } },
+				{ ItemDefinitionIndex::GLOVE_HYDRA, { ItemDefinitionIndex::INVALID, 10006, 0.0005f, -1, -1, -1, ""} },
```

Comment:
Fix your indents. (fuzion uses spaces instead of tabs)

Reply:
no idea how this one happened but quick fix !